code/modules/SCP/SCPs/SCP-173.dm

```diff
@@ -343,14 +355,23 @@ GLOBAL_LIST_EMPTY(scp173s)
 	var/area/A = get_area(src)
 	A.full_breach()
 
-// the cage
+//
+// 173 Cage
+//
+
 /obj/structure/scp173_cage
 	icon = 'icons/SCP/cage.dmi'
-	icon_state = "1"
-	name = "Empty SCP-173 Cage"
+	icon_state = "2"
```

Comment:
what are these icon state names, who named them, jeez (not issue with pr, issue with whoever named the sprites)